common/src/test/java/com/revenuecat/purchases/common/OfferingsTest.kt

```diff
@@ -39,7 +40,8 @@ class OfferingsTest {
 
         val packageToTest = getPackageJSON().createPackage(
             products,
-            "offering"
+            "offering",
+            Store.PLAY_STORE
```

Comment:
All these changes are completely unrelated, right?

Reply:
oh yeah, the test changes are left from a different approach. was waiting to update them to this one until i get sign off